test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -115,27 +118,43 @@ contract LoansForkTest is LoansTestBase {
     uint constant slippage = 1; // 1%
     uint constant BIPS_BASE = 10_000;
 
+    uint24 constant POOL_FEE_TIER = 500;
```

Comment:
best to separate oracle and swap fee tiers, as they may be different

Reply:
will do , curious why would they ever be different? wouldnt that be prone to bad prices ? since you're checking prices in one pool but swapping on another one?

Reply:
Slippage depends on trade size, but the swap fee percent is fixed. So for smalls swaps, it's possible that e.g. 500 pool is better route than 3000, even if 3000 has deeper liquidity. Also it's possible that 500 has more accurate spot price, since is arbed sooner / more often.

Also it's possible that for the oracle we'll prefer the pool with the widest liquidity, that's more resistant to manipulation, but for swaps the pool with more concentrated liquidity around current price is best.

And in any case, we're like to two swappers (of two fee tiers) enabled per pair (and FE / users would choose depending on conditions and trade size). But the oracle is likely to change less frequently.